### PR TITLE
Mock global session_status() rather than silence it

### PR DESCRIPTION
**Pull request by pachico**
I have mucked the function session_status() in sessionStart().
Rest works as used to.

### CODE REVIEW

File: tests/Storage/SessionTest.php [hunk 1]
```diff
@@ -232,7 +248,7 @@ public function testAllWithFilledStorage()
         $this->assertEquals([
             'identifier1' => 'participation1',
             'identifier2' => 'participation2',
-        ], $result);
+            ], $result);
```
Comment (HellPat) [inline tests/Storage/SessionTest.php:251]: ?

**Comment by pachico** [comment]
@waltertamboer @psren , please check.
Once we merge this Travis will give full green again.

File: tests/Storage/SessionTest.php [hunk 2]
```diff
@@ -307,7 +323,7 @@ public function testClearFilledStorage()
...
```
Comment (HellPat) [inline tests/Storage/SessionTest.php:326]: ?

File: tests/Storage/SessionTest.php (hunk 1 above)
Comment (pachico) [inline tests/Storage/SessionTest.php:251]: php-cs-fixer, seems to be.

File: tests/Storage/SessionTest.php (hunk 2 above)
Comment (pachico) [inline tests/Storage/SessionTest.php:326]: ditto

**Comment by waltertamboer** [comment]
Looks good to me!

File: tests/Storage/SessionTest.php (hunk 1 above)
Comment (waltertamboer) [inline tests/Storage/SessionTest.php:251]: I guess php-cs-fixer expects this:

``` php
$this->assertEquals(
    [
        'identifier1' => 'participation1',
        'identifier2' => 'participation2',
    ],
    $result
);
```

Comment (pachico) [inline tests/Storage/SessionTest.php:251]: Not sure but I wouldn't spend much time on this ;)

Comment (HellPat) [inline tests/Storage/SessionTest.php:251]: yes the above is correct. We can fix it before we tag.

**Pull request merged**